libkbfs/ops.go

```diff
@@ -190,8 +191,8 @@ func (co *createOp) String() string {
 	return res
 }
 
-func (co *createOp) CheckConflict(renamer ConflictRenamer, mergedOp op) (
-	crAction, error) {
+func (co *createOp) CheckConflict(renamer ConflictRenamer, mergedOp op,
```

Comment:
Nit: maybe it would be better to have `CheckConflict` take an `EntryType` instead of a bool? Not sure, though...what do you think?